src/modules/mim/daemonconfiguration.json

```diff
@@ -0,0 +1,143 @@
+{
+    "name": "DaemonConfigurationModel",
+    "type": "mimModel",
+    "contents": [
+      {	
+        "name": "DaemonConfiguration",
+        "type": "mimComponent",
+        "contents": [
+          {
+            "name": "daemons",
+            "type": "mimObject",
+            "desired": false,
+            "schema": {
+              "type": "array",
+              "elementSchema": {
+                "type": "object",
+                "fields": [
+                  {
+                    "name": "name",
+                    "schema": "string"
+                  },
+                  {
+                    "name": "state",
+                    "schema": {
+                      "type": "enum",
+                      "valueSchema": "integer",
+                      "enumValues": [
+                        {
+                          "name": "unknown",
+                          "enumValue": 0
+                        },
+                        {
+                          "name": "running",
+                          "enumValue": 1
+                        },
+                        {
+                          "name": "failed",
+                          "enumValue": 2
+                        },
+                        {
+                          "name": "exited",
+                          "enumValue": 3
+                        },
+                        {
+                          "name": "dead",
+                          "enumValue": 4
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "name": "status",
+                    "schema": {
+                      "type": "enum",
+                      "valueSchema": "integer",
+                      "enumValues": [
+                        {
+                          "name": "unknown",
+                          "enumValue": 0
+                        },
+                        {
+                          "name": "enabled",
+                          "enumValue": 1
+                        },
+                        {
+                          "name": "enabledRuntime",
+                          "enumValue": 2
+                        },
+                        {
+                          "name": "disabled",
+                          "enumValue": 3
+                        },
+                        {
+                          "name": "static",
+                          "enumValue": 4
+                        },
+                        {
+                          "name": "indirect",
+                          "enumValue": 5
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "name": "daemonCommands",
```

Comment:
>daemonCommands [](http://example.com/codeflow?start=21&length=14)

Suggested: "desiredDaemonConfigfuration"

Reply:
If we rename the component and model adding "systemd" in names, then we can use for the 2 objects respectively:

"daemonConfiguration"
"desiredDaemonConfiguration"